tealish/nodes.py

```diff
@@ -22,7 +22,8 @@
 from .scope import Scope, VarType
 
 LITERAL_INT = r"[0-9]+"
-LITERAL_BYTES = r'"(.+)"'
+LITERAL_BYTE_STRING = r'"(.+)"'
+LITERAL_BYTE_HEX = r"0x([a-zA-Z0-9]+)"
```

Comment:
Should it be limited to `a-fA-F0-9` to prevent invalid hex values?

Reply:
`^(0x)?[a-fA-F0-9]+$`

Reply:
Yes